extensions/wrapper/clients/typescript-client/package.json

```diff
@@ -49,5 +49,8 @@
         "typescript": "^4.9.3",
         "vite": "^4.2.0",
         "vite-plugin-dts": "^2.2.0"
+    },
+    "dependencies": {
+        "zod": "^3.22.4"
     }
```

Comment:
Comment: for validating three fields, this is overkill.

Reply:
but I like validation 😢

Reply:
also its 4 fields